src/process-events.c

```diff
@@ -502,17 +485,18 @@ int BPF_KPROBE_SYSCALL(kprobe__sys_execve_4_11,
 
     // create the event early so it can be re-used to save stack space
     telemetry_event_t sev = {0};
-    ptelemetry_event_t ev = &sev;
 
-    u64 id = enter_exec_4_11(SP_EXECVE, ctx, ppid, luid, ev);
+    u64 id = enter_exec_4_11(SP_EXECVE, ctx, ppid, luid, &sev);
```

Comment:
I've had issues with this and the verifier a lot in the past, which is why there's that janky allocation and pointer assignment. But since you tested it, I'm willing to say it was probably an issue with older verifiers we don't need to worry about any more.